lib/service/modules/roleStore.js

```diff
@@ -0,0 +1,139 @@
+/**
+ * Copyright (c) 2018 Kinvey Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an 'AS IS' BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+const BaseStore = require('./baseStore');
```

Comment:
I'm actually thinking of changing that rule, because I don't think it makes sense in all contexts.  Since not part of this PR, let's ignore for now and discuss it.